src/backend/index.js

```diff
@@ -104,7 +102,7 @@ feedQueue.on('drained', loadFeedsIntoQueue);
  * and save to Redis
  */
 feedQueue.on('failed', job =>
-  invalidateFeed(job.data).catch(error => logger.error({ error }, 'Unable to invalidate feed'))
+  invalidateFeed(job.data.id).catch(error => logger.error({ error }, 'Unable to invalidate feed'))
```

Comment:
Wouldn't this mean it's unable to validate the feed? 

am I wrong thinking this is a double negative?

Reply:
This is like saying:

```js
try {
  // Try and add an 'invalid' key for this feed in Redis
  await invalidateFeed(job.data.id);
} catch(error) {
  // Something went wrong trying to set the invalid key in Redis, report it and move on
 logger.error({ error }, 'Unable to invalidate feed'); 
}
```

However, when you have a callback, you can't mix callbacks and async functions, so we're using a `.catch()` to deal with the error on the Promise we get from Redis.